astropy/table/table.py

```diff
@@ -1618,12 +1662,58 @@ def add_columns(self, cols, indexes=None, copy=True, rename_duplicate=False):
               2 0.2 1.2  y
               3 0.3 1.3  z
 
+        Add unnamed columns or mixin objects at the end of the table using default names::
+            >>> t = Table([[1, 2, 3], [0.1, 0.2, 0.3]], names=('a', 'b'))
```

Comment:
This is the Table initialisation used in the previous examples for add_columns, so I used it as it is. Should I change it too?